__tests__/establishment-trading-name.test.js

```diff
@@ -0,0 +1,21 @@
+import EstablishmentTradingName from "../pages/establishment-trading-name";
+import { shallow } from "enzyme";
+import renderer from "react-test-renderer";
+import * as emotion from "emotion";
+import { createSerializer } from "jest-emotion";
+
+expect.addSnapshotSerializer(createSerializer(emotion));
+
```

Comment:
With the unit testing we did yesterday could we add some more tests here? To validate that the field receives the correct props (error message + default value)